run.sh

```diff
@@ -5,5 +5,4 @@ cd $(dirname $0)
 tools/install_spark.sh
 make backend
 . ~/.kiterc
-$SPARK_HOME/bin/spark-submit \
-  target/scala-2.12/lynxkite-0.1-SNAPSHOT.jar
+spark-submit --conf 'spark.driver.extraJavaOptions=-Dplay.http.secret.key=$KITE_APPLICATION_SECRET' target/scala-2.12/lynxkite-0.1-SNAPSHOT.jar
```

Comment:
Did you add this flag by accident?
```suggestion
spark-submit target/scala-2.12/lynxkite-0.1-SNAPSHOT.jar
```